static/src/javascripts/bootstraps/enhanced/main.js

```diff
@@ -29,6 +27,8 @@ define([
     sport
 ) {
     return function () {
+        var raven = window.guardian.app.raven;
```

Comment:
don't we want to have a `raven` module similar to `config`? This to avoid touching the `window` object.